Paper.tex

```diff
@@ -171,7 +171,7 @@ \subsection{Which History?}
 The scheme we use in order to generate consensus is a simplified version of the GHOST protocol introduced by \cite{cryptoeprint:2013:881}. This process is described in detail in section \ref{ch:ghost}.
 
 Sometimes, a path follows a new protocol from a particular height (block number).
-This document describes one version of the protocol, namely the \textit{Berlin} version defined by \cite{BeikoBerlin}.
+This document describes the \textit{London}, \textit{Arrow Glacier}, and \textit{Gray Glacier} versions of the protocol.\footnote{Note that the \textit{Arrow Glacier} and \textit{Gray Glacier} forks only affected the \hyperlink{difficulty_bomb}{"difficulty bomb"} described in section \ref{subsec:The_Block}.}
```

Comment:
I decided to point the Arrow Glacier and Gray Glacier to the same branch (london), as this was done for the Muir Glacier fork w.r.t Istanbul.

These could be separate PRs/branches, however.

Reply:
Yeah, it's fine to do them in one go with the London upgrade.